resources/public/api/conf/1.0/retrieve_se_bsas.raml

```diff
@@ -0,0 +1,26 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatBsasId
+  - errors.formatAdjustedStatus
+  - errors.ruleNotSelfEmployment
+  - errors.ruleNoAdjustmentsMade
+  - errors.clientOrAgentNotAuthorised
+  - errors.notFound
```

Comment:
The NOT_FOUND in the spec says `A calculation with this BSAS ID cannot be found` - do we want the generic or specific description?